src/server/utils/tx-manager/queueMongo.js

```diff
@@ -178,14 +178,14 @@ export default class queueMongo {
     try {
       if (this.queue.length > 0) {
         nextTr = this.queue.shift()
-
         const walletNonce = await this.getWalletNonce(nextTr.addresses)
         if (walletNonce) {
           nextTr.cb({ nonce: walletNonce.nonce, address: walletNonce.address })
+        } else {
+          this.queue.splice(0, 0, nextTr)
```

Comment:
why not push?unshift?

Reply:
why not also exception?

Reply:
I want to insert it into the begin of the queue. yes we can use unshift as well as a splice.
If we add unshift to the catch can get a problem with the infinity loop. example:
1 get from queue
2 run
3 catch and return to the queue
4 go to `1`